src/encoded/static/scss/encoded/modules/_immune_cells.scss

```diff
@@ -0,0 +1,238 @@
+$bubbleSize: 13px;
+
+$mesoderm: #008000;
+$ectoderm: #800080;
+$stem: #edcf09;
+$unlabeled: #808080;
+$endoderm: blue;
+
+$mesoderm-light: #e3fae1;
+$ectoderm-light: #e4c6f7;
+$stem-light: #fafa98;
+$unlabeled-light: #d1d1d1;
+$endoderm-light: blue;
+
+$default-active: #36586d;
+$default-hover: #f2f2f2;
+
+.node--internal text {
+    text-shadow: 0 1px 0 #fff, 0 -1px 0 #fff, 1px 0 0 #fff, -1px 0 0 #fff;
+}
+
+.link {
+    fill: none;
+    stroke: #ccc;
+    stroke-width: 2px;
+}
+
+.node-expander-collapser {
+    &:hover {
+        cursor: pointer;
+        font-weight: bold;
+    }
+}
+
+.vertical-node-graph {
+    svg {
+        max-width: 100%;
+        font-size: 0.7rem;
+
+        @media screen and (min-width: $screen-md-min) {
+            font-size: 1rem;
+        }
+    }
+
+    ellipse {
+        transition: all 50ms ease-in-out;
+        stroke: black;
+        stroke-width: 1px;
+        fill: white;
+    }
+
+    .cell-group {
+        &:hover {
+            cursor: pointer;
+        }
+    }
+
+    ellipse {
+        &:hover {
+            cursor: pointer;
+        }
+    }
+
+    .js-cell {
+        .default {
+            &.hover-class {
+                fill: $default-hover;
+            }
+        }
+
+        &.active-cell {
+            .default {
+                fill: $default-active;
+                stroke: white;
+            }
+        }
+    }
+}
+
+.hide {
+    display: none;
+}
+
+// sescc matrix-specific colors and styles
+.matrix__row-data {
```

Comment:
While 
`
matrix__row-data
`
is valid; there is the matrix pages that uses this same name. Which may cause confusion for developers trying to modify a matrix and see this in the file and no the matrix scss file.